test/activate.jl

```diff
@@ -1,10 +1,10 @@
 
-mock_run(cmd::Cmd) = cmd
+# mock_run(cmd::Cmd) = cmd
```

Comment:
I wasn't satisfied with what this was testing. I'm going to alternatives for testing activate.